clients/src/main/java/org/apache/kafka/clients/admin/CreateAclsResult.java

```diff
@@ -41,7 +41,7 @@
      * Return a map from ACL bindings to futures which can be used to check the status of the creation of each ACL
      * binding.
      */
-    public Map<AclBinding, KafkaFuture<Void>> results() {
+    public Map<AclBinding, KafkaFuture<Void>> value() {
```

Comment:
It seems kind of weird to talk about this as a singluar value since some ACL creations could fail while others succeed.  It is weird to return multiple objects and have the function name be singular.  Yes, technically we are returning a singular map, but the "values" we're really interested in are inside.